### PR TITLE
Indexes health check

### PR DESCRIPTION
**Pull request by phamann**
This introduces a new health check which checks that indexes are defined for each of the node types. It uses the REST api endpoint [`/db/data/schema/index`](http://neo4j.com/docs/2.3.0/rest-api-schema-indexes.html#rest-api-list-indexes-for-a-label) to list all currently set indexes. It will tell you the exact node types if found to be missing.

/cc @Financial-Times/next-myft

### CODE REVIEW

File: server/v2/lib/api-availability.js [hunk 1]
```diff
@@ -32,7 +31,7 @@ function checkApiAvailability() {
 }
 
 checkApiAvailability();
-setInterval(checkApiAvailability, 1000 * 60);
+setInterval(checkApiAvailability, 1000 * 60 * 60);
```
Comment (adgad) [inline server/v2/lib/api-availability.js]: Why the increase to an hour?

File: server/v2/lib/indexes.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		lastCheckOk = true;
+		lastCheckTime = currentDate;
+		panicGuide = 'Create the missing indexes.';
+		lastCheckOutput = 'Successfully retrieved database indexes';
+		panicGuide = `Don't panic`;
+	});
+}
+
+checkIndexes();
+setInterval(checkIndexes, 1000 * 60);
```
Comment (adgad) [inline server/v2/lib/indexes.js]: I presume this one was meant to be an hour...

**Comment by ironsidevsquincy** [comment]
Maybe worth using `https://github.com/Financial-Times/n-health`? Wraps up a lot of common functionality

File: server/v2/lib/indexes.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+			name: "Next MyFT API indexes",
+
+			ok: lastCheckOk,
+			checkOutput: lastCheckOutput,
+			lastUpdated: lastCheckTime,
+			panicGuide: panicGuide,
+
+			severity: 2,
+			businessImpact: "Next MyFT api calls may slow down and error.",
+			technicalSummary: "Makes an API call which checks all required Node indexs are defined."
```
Comment (wheresrhys) [inline server/v2/lib/indexes.js]: typo

**Pull request merged**